controllers/rcj_soccer_team_yellow/rcj_soccer_robot.py

```diff
@@ -38,7 +36,7 @@ def parse_supervisor_msg(self, packet: str) -> dict:
                     'B1': {'x': 0.0, 'y': 0.2, 'orientation': 1},
                     'B2': {'x': 0.4, 'y': -0.2, 'orientation': 1},
                     ...
-                    'ball': {'x': -0.7, 'y': 0.3},
+                    'ball': {'x': -0.7, 'y': 0.3}
```

Comment:
Was the removal of the comma from the end of this line on purpose?

Reply:
Thanks, this was not intentional.